docs/Ansible.md

```diff
@@ -4,47 +4,23 @@ There are many variables that can be configured to tune the setup.
 Some are useful only in development environment and some are not really meant
 to be changed.
 
-| Variable                       | Default value | Description                                                                                                                           |
-| -----------------------------  | ------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
-| v2v_transport_methods          |               | List of transport methods to configure on the conversion host. Valid values: `vddk`, 'ssh`.                                           |
-| v2v_max_concurrent_conversions |               | Override the maximum concurrent conversions per host in ManageIQ configuration.                                                       |
-| v2v_vddk_package_name          |               | File name of the .tar.gz package with VDDK library. It is looked for in /tmp.                                                         |
-| v2v_vddk_package_url           |               | URL to the VDDK library package (will be downloaded to `/tmp`). Note that the file name must much the one in `v2v_vddk_package_name`. |
-| v2v_vddk_override              | false         | Normally the install role is not run if the plugin is already installed. To force the deployment set this variable to `true`.         |
-| v2v_ssh_private_key            |               | The private key to use to connect to the VMware host.                                                                                 |
-| v2v_checks_override            | false         | The install role does performs some compatibility checks. By setting `v2v_checks_override` to `true` one can disable them.            |
+| Variable                       | Default value | Description                                                                                                                   |
+| -----------------------------  | ------------- | ----------------------------------------------------------------------------------------------------------------------------- |
+| v2v_host_type                  |               | Platform where the conversion host is deployed. Valid values: `rhevm`, `openstack`.                                           |
+| v2v_transport_method           |               | Transport methods to configure on the conversion host. Valid values: `vddk`, 'ssh`.                                           |
```

Comment:
says `Transport methods` but if I understand correctly only one method can be configured with one playbook run